nilearn/plotting/displays.py

```diff
@@ -260,14 +261,25 @@ def _add_lines(self, line_coords, line_values, cmap, **kwargs):
                 values of the lines.
             cmap: colormap
                 colormap used to map line_values to a color.
+            vmin: float, optional, default: None
+            vmax: float, optional, default: None
+                If not None, either or both of these values will be used to
+                as the minimum and maximum values to color lines. If None are
+                supplied the maximum absolute value within the given threshold
+                will be used as minimum (multiplied by -1) and maximum
+                coloring levels.
             kwargs: dict
                 additional arguments to pass to matplotlib Line2D.
         """
         abs_line_values_max = np.abs(line_values).max()
-        norm = colors.Normalize(vmin=-abs_line_values_max,
-                                vmax=abs_line_values_max)
+        if vmin is None:
+            vmin = -abs_line_values_max
```

Comment:
> I think that vmax should be set first, and then vmin set to -vmax if none.

Agreed. That means:
* vmin and vmax are None: current behaviour, `vmax = np.max(np.abs(edge_data))`, `vmin = -vmax`
* vmin is None, vmax set: `vmin = -vmax`
* vmin set, vmax is None: `vmax = np.max(np.abs(edge_data))`, leave vmin as provided
* vmin and vmax set: leave vmax and vmin as provided

> However, @GaelVaroquaux , what happens in your case if vmin is positive and vmax is None?

See above. About these weird edge cases, I would be in favor of not trying to catch them all and just letting the matplotlib code do whatever it does in these cases.